Hunting Queries/CloudAppEvents/SetPolicyConfigInCloudAppEvents.yaml

```diff
@@ -0,0 +1,32 @@
+id: fcd4d774-a0c2-4d12-9e9f-f51dfc310873
+name: 
+description: |
+  "We search for any action type with high frequency that involves adding/modifying/removing something in cloud app policies. We want to see where the properties are modified such that the old value and new value are different, and exclude DisplayName as changing Display Name is a minor property change."
+severity: Medium
+requiredDataConnectors:
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - CloudAppEvents
+queryFrequency: 1h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Privilege Escalation
+query: |
+  CloudAppEvents
+    | where ActionType in ("Update policy.", "Add policy.", "Remove-CrossTenantAccessPolicy", "Add policy to service principal.", "Write PolicyAssignments", "Update authorization policy.", "Delete policy.", "Add owner to policy.", "Write PolicyExemptions", "Remove-LabelPolicy")
+    | mv-expand ActivityObjects
+    | where ActivityObjects.Name != "DisplayName"
+    | summarize Count = count() by AccountDisplayName, IPAddress, ActionType, ActivityType
+entityMappings:
+  - entityType: Account

```

Comment:
Lets check what strong identifiers are present for Account entity and map all here eg. UPNSuffix, Name, Full Name